src/java/picard/reference/ExtractSequences.java

```diff
@@ -50,9 +50,18 @@
  * @author Tim Fennell
  */
 @CommandLineProgramProperties(
-        usage = "Extracts one or more intervals described in an interval_list file " +
-                "from a given reference sequence and writes them out in FASTA format. Requires a fasta index " +
-                "file to be present.",
+        usage = "Extracts one or more intervals described in an interval_list file from a given reference sequence" +
```

Comment:
Do we have any info on how the extracted sequences are named? I.e. pattern for indicating contig/start/stop?